### PR TITLE
Bug 1693489 -  Update Glean SDK development and Glean book front page content

### PR DESCRIPTION
**Pull request by brizental**
I tried to make the content on the front page of the user book a bit less SDK specific with the future addition of Glean.js documentation in mind.

I sent a third commit where I mention both Glean clients in the user book's front page. I made it into a separate commit on purpose, after I was done and read it, I wondered if we should hold off on mentioning it in the front page (since all the rest of the book is still heavily SDK specific).

### CODE REVIEW

File: docs/user/README.md [hunk 1]
```diff
@@ -1,44 +1,67 @@
-# Glean SDK
+# Introduction
 
-![Glean logo](glean.jpeg)
+Glean is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
```
Comment (badboy) [inline docs/user/README.md:3]: While we're here make it lowercase telemetry:
```suggestion
Glean is a modern approach for a telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
```

File: docs/user/README.md [hunk 2]
```diff
@@ -1,44 +1,67 @@
...
+## Contact
 
-The `Glean SDK` is a modern approach for a Telemetry library and is part of the [Glean project](https://docs.telemetry.mozilla.org/concepts/glean/glean.html).
+To contact the Glean team you can:
 
-To contact us you can:
 - Find us in the [#glean channel on chat.mozilla.org](https://chat.mozilla.org/#/room/#glean:mozilla.org).
-- To report issues or request changes, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
 - Send an email to *glean-team@mozilla.com*.
 - The Glean SDK team is: *:janerik*, *:dexter*, *:travis*, *:mdroettboom*, *:gfritzsche*, *:chutten*, *:brizental*.
```
Comment (badboy) [inline docs/user/README.md:11]: Should we update this? Georg's out for a while now :(

File: docs/user/README.md [hunk 3]
```diff
@@ -1,44 +1,67 @@
...
 
-The source code is available [on GitHub](https://github.com/mozilla/glean/).
+![Glean logo](glean.jpeg)
```
Comment (badboy) [inline docs/user/README.md:13]: What was the reason for moving the logo down here?
IMO it was good as the big splash to start with.

Plus now the contact info is front and center and I'm not sure that's the best way

File: docs/user/README.md [hunk 4]
```diff
@@ -1,44 +1,67 @@
...
+
+### [Glean.js](https://github.com/mozilla/glean.js)
+
+Glean.js is an implementation of Glean in **Javascript**. Currently, it only has support for usage in web extensions.
+
+For development documentation on `Glean.js`, refer to [the Glean.js development documentation](https://github.com/mozilla/glean.js/tree/main/docs).
+
+To report issues or request changes on the Glean SDK, file a bug in [Bugzilla in Data Platform & Tools :: Glean.js](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean.js&priority=P4&status_whiteboard=%5Btelemetry%3Aglean-js%3Am%3F%5D).
+
+> **Note** Glean.js is still on the MVP stage and does not provide all the features the Glean SDK does. Feature parity will be worked on after initial validation. Do not hesitate to file bugs using the link above if you want to use Glean.js and is missing some key Glean feature.
```
Comment (badboy) [inline docs/user/README.md:35]:
```suggestion
> **Note** Glean.js is still in development and does not provide all the features the Glean SDK does. Feature parity will be worked on after initial validation. Do not hesitate to [file a bug][gleanjs-bugs] if you want to use Glean.js and is missing some key Glean feature.

[gleanjs-bugs]: https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean.js&priority=P4&status_whiteboard=%5Btelemetry%3Aglean-js%3Am%3F%5D
```

File: docs/user/README.md [hunk 5]
```diff
@@ -1,44 +1,67 @@
...
+
+To report issues or request changes on the Glean SDK, file a bug in [Bugzilla in Data Platform & Tools :: Glean: SDK](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean%3A+SDK&priority=P3&status_whiteboard=%5Btelemetry%3Aglean-rs%3Am%3F%5D).
+
+### [Glean.js](https://github.com/mozilla/glean.js)
+
+Glean.js is an implementation of Glean in **Javascript**. Currently, it only has support for usage in web extensions.
+
+For development documentation on `Glean.js`, refer to [the Glean.js development documentation](https://github.com/mozilla/glean.js/tree/main/docs).
+
+To report issues or request changes on the Glean SDK, file a bug in [Bugzilla in Data Platform & Tools :: Glean.js](https://bugzilla.mozilla.org/enter_bug.cgi?product=Data+Platform+and+Tools&component=Glean.js&priority=P4&status_whiteboard=%5Btelemetry%3Aglean-js%3Am%3F%5D).
```
Comment (badboy) [inline docs/user/README.md:33]:
```suggestion
To report issues or request changes on the Glean SDK, file a bug in [Bugzilla in Data Platform & Tools :: Glean.js][gleanjs-bugs].
```

File: docs/user/README.md [hunk 6]
```diff
@@ -1,44 +1,67 @@
...
+
+> **Note** Glean.js is still on the MVP stage and does not provide all the features the Glean SDK does. Feature parity will be worked on after initial validation. Do not hesitate to file bugs using the link above if you want to use Glean.js and is missing some key Glean feature.
+
+## Sections
+
+### [Using Glean](./user/index.html)
+
+If you want to start using Glean to report data then this is the section you should read. It explains the first steps of integrating Glean into your project, choosing the right metric type for you, debugging products that use Glean and even Glean's internal error reporting mechanism.
```
Comment (badboy) [inline docs/user/README.md:41]: I think I would turns this around slightly:

```suggestion
In this chapter we describe how to use the Glean SDK in your own libraries and applications.
It explains the first steps of integrating Glean into your project, choosing the right metric type for you, debugging products that use Glean and Glean's built-in error reporting mechanism.

If you want to start using Glean to report data then this is the section you should read.
```

**Review by badboy — COMMENTED** [review]
gave this a first look, need to re-read it as a whole.

I would suggest to use [semantic line breaks](https://sembr.org/), that will make subsequent editing easier (or at least the diffs easier to read).

File: docs/user/README.md (hunk 3 above)
Comment (brizental) [reply docs/user/README.md:13]: Yeah, I had put the contact before the `## Sections` title, but I was torn... I can move it back there, or somewhere else. The truth is I didn't know where to put it :D

File: docs/user/README.md (hunk 2 above)
Comment (mdboom) [reply docs/user/README.md:11]: Or we can move his name to a "contributors" or "emeritus" section...

File: docs/user/README.md (hunk 4 above)
Comment (brizental) [reply docs/user/README.md:35]: I seems the link checked is not to happy about the link alias :/

File: docs/user/README.md (hunk 3 above)
Comment (Dexterp37) [reply docs/user/README.md:6]: Side-note: do we want the logo with a white or transparent background to be shown here?

Comment (brizental) [reply docs/user/README.md:6]: > Side-note: do we want the logo with a white or transparent background to be shown here?

I thought about this. The black adds a bit of drama, which I appreciate. IMO, we should keep it black.

File: docs/user/README.md (hunk 5 above)
Comment (brizental) [reply docs/user/README.md]: See https://github.com/mozilla/glean/pull/1528#discussion_r585379503,do you know a workaround for the link checker issue?

File: docs/user/README.md (hunk 2 above)
Comment (brizental) [reply docs/user/README.md]: "alumni" maybe? 

That would not fit in the `# Contact` section though, IMO. How about we remove his name here and file a bug for adding a `contributors` section somewhere else? Maybe the Github README? We could maybe even automate that based on commit history or something. Outside contributors might appreciate that :D

File: docs/user/README.md (hunk 3 above)
Comment (brizental) [reply docs/user/README.md:6]: (I moved the `Contact` section to right before the `License` section in this commit https://github.com/mozilla/glean/pull/1528/commits/e45e44225e54cf570489499c1c9928d2dbab72eb, still not 100% secure about its positioning)

File: docs/user/README.md (hunk 2 above)
Comment (badboy) [reply docs/user/README.md]: Let's leave it and tackle that separately.

File: docs/user/README.md (hunk 6 above)
Comment (badboy) [reply docs/user/README.md]: Works for me!

One last nitpick: You use `section` for the other parts so I think it'd be better to also say "In this section ..." here. Sorry, that was my mistake to suggest "chapter"

File: docs/user/README.md [hunk 7]
```diff
@@ -1,44 +1,91 @@
...
+> **Note** Glean.js is still in development and does not provide all the features the Glean SDK does.
+> Feature parity will be worked on after initial validation. Do not hesitate to [file a bug][gleanjs-bugs]
+> if you want to use Glean.js and is missing some key Glean feature.
+## Sections
 
-This book is specifically about **using** the `Glean SDK`.
+### [Using Glean](./user/index.html)
 
-For documentation on **contributing to and developing** the `Glean SDK`, refer to [the Glean SDK development book](../dev/).
+In this chapter we describe how to use Glean in your own libraries and applications.
```
Comment (badboy) [inline docs/user/README.md:43]:
```suggestion
In this section we describe how to use Glean in your own libraries and applications.
```

to make it consistent with the next parts. Sorry for suggesting "chapter" to begin with.

**Review by badboy — APPROVED** [review]
I'd say we should land this.
There might be a few smaller followups, but I'd rather have this out now and fixed in subsequent smaller patches.

**Pull request merged**